tests/prague/eip7692_eof_v1/eip7620_eof_create/test_subcontainer_validation.py

```diff
@@ -756,3 +756,44 @@ def test_migrated_eofcreate(eof_test: EOFTestFiller, container: Container):
     Tests migrated from EOFTests/efValidation/EOF1_eofcreate_valid_.json.
     """
     eof_test(data=container, expect_exception=container.validity_error)
+
+
+def test_dangling_initcode_subcontainer_bytes(
+    eof_test: EOFTestFiller,
+):
+    """Initcode mode EOF Subcontainer test with subcontainer containing dangling bytes."""
+    eof_test(
+        data=Container(
+            sections=[
+                returncontract_code_section,
+                Section.Container(
+                    custom_size=len(stop_sub_container.data),
```

Comment:
I think this is different from the issue found by the fuzzer: there the section size (21, 0x15) matches the content, so the error is only in the subcontainer itself. So maybe parametrize this detail?

Reply:
You're right, fixed.

What would be parameterized?  I'm afraid if we parameterize it the failure we are probing for will be missed.

Reply:
param value 1: as you have written it originally
param value 2: as the fuzzer fount it (current version)

Reply:
I.e. with and without `custom_size`.

Reply:
Parametrizing it that way just turns the custom size EOF into another basic container test, which exists as `test_simple_create_from_deployed` already.